src/api/expenses/expenses.controller.js

```diff
@@ -0,0 +1,87 @@
+const { expensesService } = require('../../services/expenses/expenses.service');
+const { usersService } = require('../../services/users/users.service');
+const { filterExpensesWithQuery } = require('../../services/helperService');
+const getAll = (req, res) => {
+  res.send(filterExpensesWithQuery(req.query));
+};
+
+const create = (req, res) => {
```

Comment:
Consider wrapping the code inside the controller into the try-catch block and returning the 500 error in the catch